boards/matek/h743-slim/init/rc.board_mavlink

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# PX4 FMUv6U specific board MAVLink startup script.
+#------------------------------------------------------------------------------
+
+# Start MAVLink on the USB port
+mavlink start -d /dev/ttyACM0
```

Comment:
The mavlink USB instance will automatically start if the board has VBUS available for detection.

Reply:
Didn't know that, thanks. I guess then we can delete `rc.board_mavlink`